### ISSUE TITLE
Kite is always started

### ISSUE THREAD

**Issue by goanpeca** [spyder-ide/spyder#10071]
I am not sure if this is a bug or a feature. I have not been using `kite` that much since it consumes too much resources, so I usually shut down the client on OSX, however spyder is always starting the client and I do not see any option to prevent this.

Thoughts @andfoy, @ccordoba12 ?

**Comment by andfoy** [comment]
The Kite team is due to send a PR to set the starting, fault tolerance and teardown if Kite. Sadly for you, but Kite will active, unless you deactivate manually (in the configuration panel that Will be implemented later on) or uninstalled

**Comment by metalogical** [comment]
I think the correct solution here is to just add a setting in the configuration panel "start Kite automatically if installed."

**Comment by ccordoba12** [comment]
You're right @metalogical. @goanpeca himself is going to take care of that.
